components/AvatarList/index.js

```diff
@@ -0,0 +1,28 @@
+import AvatarContainer from "../AvatarContainer";
+
+import styles from "./styles.module.css";
+
+const AvatarList = () => {
+  return (
+    <div className={styles.container}>
+      <div className={styles.avatarsContainer}>
+        <div className={styles.avatarColumn}>
+          <AvatarContainer>A</AvatarContainer>
```

Comment:
Nice work!

In front-end dev, we often create lists of things. We can leverage naming conventions to show how to components work together.

In this case:
`AvatarList` and `AvatarListItem`

Through naming we can get a hint of the relationship between the two.